16-synthesis.Rmd

```diff
@@ -57,9 +57,9 @@ identical(nz_name1$Name, nz_name2$Name) # check results
 
 This raises the question: which to use?
 The answer is: it depends.
-Each approach has advantages: the pipe syntax is popular and intuitive, while base R\index{R!base} tends to stable, well-known, and has minimal dependencies.
-Choosing between them is therefore largely a matter of preference.
-Our advice is to try both and see which you prefer.
+Each approach has advantages: base R\index{R!base} tends to be stable, well-known, and has minimal dependencies which is why it is often preferred for software (package) development.
```

Comment:
I wouldn't mention the base pipe as an advantage for using the tidyverse since you can use it both with base R and the tidyverse (e.g., if you must, you can write ` nz |> `[`(_$Name)`). I guess this was just a leftover argument before switching from the magrittr pipe to the newly introduced base pipe. In any case, I have made here the argument that base R is often preferred for software development while the tidyverse is often preferred for interactive programming. But feel free to change.

Reply:
Yes, that was a leftover argument. Good point Jannes and :+1: to the change.